boards/FSM/Src/Io/Io_FlowMeters.c

```diff
@@ -0,0 +1,84 @@
+#include "main.h"
+#include "Io_SharedFrequencyOnlyPwmInput.h"
+#include "Io_FlowMeters.h"
+#include <assert.h>
+
+#define PRIMARY_FLOW_METER_CHANNEL HAL_TIM_ACTIVE_CHANNEL_1
```

Comment:
I would just add a new field for this in `Context` that would be initialized in `Create`. Grouping every thing related to flow meter in a struct is better than having floating macros.

That way we can write 
```
if (htim->Channel == primary_flow_meter.htim_channel)
```
instead of
```
if (htim->Channel == PRIMARY_FLOW_METER_CHANNEL)
```